src/ObjectUtils/CoffFileTest.cpp

```diff
@@ -34,19 +34,21 @@ TEST(CoffFile, LoadDebugSymbols) {
 
   std::vector<SymbolInfo> symbol_infos(symbols_result.value().symbol_infos().begin(),
                                        symbols_result.value().symbol_infos().end());
-  EXPECT_EQ(symbol_infos.size(), 53);
+  EXPECT_EQ(symbol_infos.size(), 35);
 
-  SymbolInfo& symbol_info = symbol_infos[0];
+  SymbolInfo& symbol_info = symbol_infos[4];
```

Comment:
Is this an `extern "C"` symbol?

Reply:
I don't think so, why do you ask?

Reply:
I was thinking of adding a test for the case when DW_AT_LINKAGE_NAME is not present.

Reply:
As discussed, getName will fallback to ShortName if LinkageName is not present.